sysgen/sysgen.go

```diff
@@ -465,27 +507,26 @@ func generateArg(
 			val = "0"
 			skipSyscall(fmt.Sprintf("missing const %v", a[0]))
 		}
-		fmt.Fprintf(out, "ConstType{%v, TypeSize: %v, Val: uintptr(%v)}", common(), size, val)
+		fmt.Fprintf(out, "ConstType{%v, TypeSize: %v, Endianness: %v, Val: uintptr(%v)}", common(), size, endianness, val)
 	case "strconst":
 		canBeArg = true
 		if want := 1; len(a) != want {
 			failf("wrong number of arguments for %v arg %v, want %v, got %v", typ, name, want, len(a))
 		}
 		fmt.Fprintf(out, "PtrType{%v, Dir: %v, Type: StrConstType{%v, Val: \"%v\"}}", common(), fmtDir("in"), common(), a[0]+"\\x00")
-	case "int8", "int16", "int32", "int64", "intptr":
+	case "int8", "int16", "int32", "int64", "intptr", "int8be", "int16be", "int32be", "int64be", "intptrbe":
```

Comment:
whenever I will see int8be in syscall description, I will say s/int8be/int8/
int8be is confusing, drop it